app/Http/Controllers/Api/VideoAnnotationController.php

```diff
@@ -240,6 +240,7 @@ public function store(StoreVideoAnnotation $request)
             $queue = config('videos.track_object_queue');
             Queue::pushOn($queue, new TrackObject($annotation, $request->user()));
             Cache::increment(TrackObject::getRateLimitCacheKey($request->user()));
+            $annotation->trackedAnnotationRate = $currentJobs / (float) ($maxJobs-1);
```

Comment:
Please update the test for this.

You could also use this to prevent possible rounding/precision errors:

```suggestion
            $annotation->trackingJobLimitReached = $currentJobs === ($maxJobs - 1);
```